OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
@penev92 In RA it can take out a conyard. In D2k it definitely can't. Dunno TD.

@fadyanwar I was a really good mood. http://pastebin.com/i39ndBXJ Thank you for ruining it with a spoonfeeding request about something which enough people told you already.